rpc_jobs/rpc_gating_project_builds.yml

```diff
@@ -0,0 +1,29 @@
+- job:
+    name: 'RPC-Gating-Project-Builds'
+    project-type: workflow
+    concurrent: true
+    triggers:
+      - github-pull-request:
+          org-list:
+            - rcbops
+          github-hooks: true
+          trigger-phrase: '.*recheck_all.*|.*recheck_project_builds.*'
+          only-trigger-phrase: false
+          auth-id: "github_account_rpc_jenkins_svc"
+          status-context: 'CIT/project_builds'
```

Comment:
Ahh, unless the whole clause is omitted in which case, PRs for all branches will be tested, which is fine for this repo, as all PRs are against master.